CMakeLists.txt

```diff
@@ -7,4 +7,6 @@ set (CoreCVS_VERSION_MINOR 0)
 add_subdirectory(core)
 add_subdirectory(test-core)
 
+add_dependencies(test-core corecvs)
```

Comment:
No-no! It's done via one command:
`   target_link_libraries(corecvs_tests corecvs)`
and only! Indeed this line above adds include dirs. and the linking with its library.